src/Orchard.Cms.Web/Modules/Orchard.OpenId/Models/OpenIdScope.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Orchard.OpenId.Models
+{
+    public class OpenIdScope
```

Comment:
@PinpointTownes Can you provide a case of use? I imagine for what it is, bu actually I haven't seen an use case.